src/ch/imedias/rsccfx/view/RsccPresenter.java

```diff
@@ -6,7 +6,10 @@
   private final Rscc model;
   private final RsccView view;
 
-  /** Javadoc comment here. */
+  // FIXME: Use this as main Presenter and base all of the presenters around it on this
```

Comment:
We renamed it in the "basicLayoutLogic" branch so we won't mix anything up.
So that everyone knows: We called it "RsccHomePresenter.java" to kind of show where we start. I will add a todo into it that we will change it back.
@JenniferMue @pwigger